### PR TITLE
Graduate KubeProxyDrainingTerminatingNodes to GA

### PR DESCRIPTION
**Pull request by alexanderConstantinescu**
Enhancement: https://github.com/kubernetes/enhancements/issues/3836

Original implementation PR: https://github.com/kubernetes/kubernetes/pull/116470

Original doc PR: https://github.com/kubernetes/website/pull/45678

Superseedes: https://github.com/kubernetes/website/pull/47030

/sig network
/cc @aojea

### CODE REVIEW

**Comment by Shubham82** [comment]
Thanks @alexanderConstantinescu
/lgtm

File: content/en/docs/reference/command-line-tools-reference/feature-gates/kube-proxy-draining-terminating-nodes.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ stages:
   - stage: beta
     defaultValue: true
     fromVersion: "1.30"
+    toVersion: "1.31"
```
Comment (dipesh-rawat) [inline content/en/docs/reference/command-line-tools-reference/feature-gates/kube-proxy-draining-terminating-nodes.md:16]: The `toVersion` need correction to reflect the last version for the previous stage.

```suggestion
    toVersion: "1.30"
```

Comment (alexanderConstantinescu) [reply content/en/docs/reference/command-line-tools-reference/feature-gates/kube-proxy-draining-terminating-nodes.md]: ACK, I've applied the change

Comment (Shubham82) [reply content/en/docs/reference/command-line-tools-reference/feature-gates/kube-proxy-draining-terminating-nodes.md]: I missed it, thanks @dipesh-rawat for the suggestion.

**Comment by kundan2707** [comment]
Thanks for change
/lgtm

**Review by dipesh-rawat — COMMENTED** [review]
/approve

**Pull request merged**